src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -62,6 +62,10 @@ export default class CourtOrderPoa extends Vue {
   @Prop({ default: true })
   readonly displaySideLabels: boolean
 
+  /** Wether court order number is required regardless plan of arrangement. */
+  @Prop({ default: false })
+  readonly courtOrderNumberRequired: boolean
```

Comment:
Under what conditions is Court Order Number required?

Is this the validation for https://github.com/bcgov/entity/issues/7473  ?

Reply:
No, it's for bcgov/entity#4388. The court order notation should have should have the court order number required. This was confirmed today by Linda.

Reply:
ok

Reply:
Does "ok" mean "approved"? 😄